src/components/BaseLayout.tsx

```diff
@@ -3,28 +3,33 @@ import classnames from "classnames";
 
 interface Props {
   title?: string | ReactNode;
-  controls?: ReactNode | null;
+  controls?: ReactNode;
+  customHeader?: ReactNode;
```

Comment:
This is creating an opinionated interface, namely when passing a `title`, the `customHeader` will be ignored. Maybe a better approach than making this `BaseLayout` cover all cases is to introduce a `CustomLayout` which receives the `header` component instead of `title` and `controls`?

Reply:
Also we can make the `title` prop mandatory and the single exception of `Login` can also use the `CustomLayout`?